crates/rbuilder/src/beacon_api_client/mod.rs

```diff
@@ -0,0 +1,88 @@
+use beacon_api_client::{mainnet::Client as bClient, Error, Topic};
+use mev_share_sse::client::EventStream;
+use reth::rpc::types::beacon::events::PayloadAttributesEvent;
+use serde::Deserialize;
+use std::{collections::HashMap, fmt::Debug};
+use url::Url;
+
+#[derive(Deserialize, Clone)]
+#[serde(try_from = "String")]
+pub struct Client {
+    inner: bClient,
+}
+
+impl Debug for Client {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("Client").finish()
+    }
+}
+
+impl Default for Client {
+    fn default() -> Self {
+        Self {
+            inner: bClient::new(Url::parse("http://localhost:8000").unwrap()),
```

Comment:
don't use hidden unwraps

Reply:
can you put "http://localhost:8000" as a const and replace it in the 3 places you use it?